csharp/test/Microsoft.AI.MachineLearning.Tests.DotNetCore3_0/Microsoft.AI.MachineLearning.Tests.DotNetCore3_0.csproj.pp

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do you know what's the main difference between .csproj and .csproj.pp ? Also .config and .config.pp

Reply:
Yep - the pp files are preprocessed files. They are consumed by the azure pipeline and they produce the non-pp file. the project needs to be updated needs to refer to the exact package produced by the build to verify it. It adds in the version number.